src/pages/word-duel.jsx

```diff
@@ -37,44 +38,46 @@ export default function WordDuel() {
         return true;
     }
 
-    /* 
-    TODO: These are very rudamentary rule but they are sufficient for now 
-    There are some corner cases that can occur when there are repeated letters in a guess
-    Case 1: Guess contains multiples of a letter where only 1 is in the word and both "miss"
-        Actual: Both letters are yellow
-        Expected: One letter should be yellow (the first one) and the other should be gray
-    Case 2: Guess contains multiples of a letter where only 1 is in the word where 1 "hits" and 1 "misses"
-        Actual: "Miss" is yellow and "hit" is green
-        Expected: "Miss" is gray and "hit" is green
-    Need to devise a way to make sure each letter is only used once when evaluating a guess
-    */
     function evaluateGuess(guess) {
-        const evaluation = guess.split("").map((letter, idx) => {
-            if (letter === word[idx]) {
-                return EvaluationColor.Green;
-            }
+        const evaluation = new Array(word.length).fill(EvaluationColor.Gray);
```

Comment:
I know there's an easier way to write this, but since _this works_ I'd say we can brainstorm later.  Maybe we can extract a couple of these functions to some shared library and write some tests.

Reply:
Yeah I know it's not pretty but it gets the job done in its own convoluted way. Would absolutely love to figure out a less rough way to do this.